### PR TITLE
add the rest of the html global attributes

### PR DESCRIPTION
**Pull request by mpacer**
This should cover everything from the current [w3c spec](https://dev.w3.org/html5/spec-preview/global-attributes.html)

### CODE REVIEW

File: lib/completions-jsx.coffee [hunk 1]
```diff
@@ -77,6 +77,18 @@ globalAttributes: [
...
+        { name: 'accesskey'}
+        { name: 'contenteditable'}
+        { name: 'contextmenu'}
+        { name: 'dir'}
+        { name: 'draggable'}
+        { name: 'dropzone'}
+        { name: 'hidden'}
+        { name: 'lang'}
+        { name: 'spellcheck'}
+        { name: 'tabindex'}
```
Comment (rgbkrk) [inline lib/completions-jsx.coffee]: Some of these need to be camelCased, like `tabindex` ➡️  `tabIndex`. See https://facebook.github.io/react/docs/dom-elements.html for a full description.

File: lib/completions-jsx.coffee [hunk 2]
```diff
@@ -77,6 +77,18 @@ globalAttributes: [
 	{ name: 'data-'}
 	{ name: 'aria-'}
 	{ name: 'style'}
+        { name: 'accesskey'}
+        { name: 'contenteditable'}
```
Comment (rgbkrk) [inline lib/completions-jsx.coffee]: `contentEditable`

File: lib/completions-jsx.coffee (hunk 1 above)
Comment (rgbkrk) [reply lib/completions-jsx.coffee]: In fact, it looks like there's an exhaustive list including the casing here: https://facebook.github.io/react/docs/dom-elements.html#all-supported-html-attributes

File: lib/completions-jsx.coffee [hunk 3]
```diff
@@ -77,6 +77,18 @@ globalAttributes: [
...
+        { name: 'contextmenu'}
+        { name: 'dir'}
+        { name: 'draggable'}
+        { name: 'dropzone'}
+        { name: 'hidden'}
+        { name: 'lang'}
+        { name: 'spellcheck'}
```
Comment (rgbkrk) [inline lib/completions-jsx.coffee]: `spellCheck`

File: lib/completions-jsx.coffee [hunk 4]
```diff
@@ -77,6 +77,18 @@ globalAttributes: [
 	{ name: 'data-'}
 	{ name: 'aria-'}
 	{ name: 'style'}
+        { name: 'accesskey'}
+        { name: 'contenteditable'}
+        { name: 'contextmenu'}
```
Comment (rgbkrk) [inline lib/completions-jsx.coffee]: `contextMenu`

File: lib/completions-jsx.coffee [hunk 5]
```diff
@@ -77,6 +77,18 @@ globalAttributes: [
 	{ name: 'data-'}
 	{ name: 'aria-'}
 	{ name: 'style'}
+        { name: 'accesskey'}
```
Comment (rgbkrk) [inline lib/completions-jsx.coffee]: `accessKey`

**Comment by mpacer** [comment]
And dropZone isn't even a thing! Fixes on there way

**Pull request merged**